services/button.md

```diff
@@ -13,24 +13,28 @@ A push-button, which returns to inactive position when not operated anymore.
 
 Indicates whether the button is currently active (pressed).
 
+    rw hold_time: u16 { absolute_min=500 } ms @ 0x80
```

Comment:
```suggestion
    rw hold_time = 1000: u16 { absolute_min=500 } ms @ 0x80
```

Reply:
also the 500 absolute min may be a bit low

Reply:
i would say at least 1500 for hold, which allows for longer holds.

Reply:
sorry, actually I meant too restrictive; maybe 100ms